src/client/src/ui/analytics/components/LastPosition.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import numeral from 'numeral'
+import { LastPositionWrapper, LastPositionStyle, USDspan } from './styled'
+
+interface LastPositionProps {
+  lastPos?: number | undefined
+}
+
+interface FormattedLastPosition {
+  lastPosition: string | undefined
+  color: string
+}
+
+const formatLastPosition: (lastPos: number) => FormattedLastPosition = (lastPos: number) => {
+  let lastPosition = numeral(lastPos).format()
+  let color = ''
+  if (lastPos > 0) {
```

Comment:
would a ternary simplify this? `const color = lastPos > 0 ? 'green' : lasPos < 0  ? ' red' : ''`